deps/openlibm.version

```diff
@@ -1,2 +1,2 @@
-OPENLIBM_BRANCH=v0.4.1
-OPENLIBM_SHA1=3ee2a6e6a9fc68cbf2ab00f830b9059215e40b06
+OPENLIBM_BRANCH=master
```

Comment:
I think I understand it.  The confusing part is that with `git`, a branch name and a tag name are often interchangeable.  I've tagged new releases of `openlibm` and `openspecfun`, and pushed a new commit with those tag names in the `_BRANCH` variables.